bitnami/argo-workflows/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
-  version: 12.0.0
+  version: 12.0.1
 - name: mysql
   repository: https://charts.bitnami.com/bitnami
   version: 9.4.1
 - name: common
   repository: https://charts.bitnami.com/bitnami
-  version: 2.1.1
-digest: sha256:423faba9bef43ffb8dacffbf981417acff4a989465efeaad16ba748510f322cf
-generated: "2022-10-31T10:03:43.049486286Z"
+  version: 2.1.2
+digest: sha256:0f61e68267ea3a6858555abe6489ee24a29f0c562641a5543b29d627612c4244
+generated: "2022-11-02T17:52:12.969845+01:00"
```

Comment:
This is okay but you may probably want to dismiss this change from the diff in order to avoid conflicts with the master branch.